.gitignore

```diff
@@ -0,0 +1 @@
+/.idea
```

Comment:
why we need this file?

Reply:
we dont need this, please tell how to remove this?